resources/views/footer-copyright.blade.php

```diff
@@ -5,27 +5,26 @@
 ])
 
 <div {{ $attributes->merge(['class' => 'flex flex-col py-6 space-y-2 font-semibold text-sm text-theme-secondary-700 sm:flex-row sm:space-y-0 sm:space-x-1'])}}>
-    <span class="whitespace-nowrap">
+    <span class="break-words">
```

Comment:
did you check this on MSQ to ensure it doesn't cause trouble there?

Reply:
I did, and it doesn't specifically breaks MSQ layout, but I have a tiny issue where I can't apply proper spacing "between" the text on < 640px (both on Payvo & MSQ) ; 

![](https://gyazo.com/53674a9467be567e92073cb3b3cb56d8.png)

Other than that, it does looks good and as shown in the designs 🤔

Reply:
@Highjhacker that's because you changed the nesting, so the `space-y-2` no longer applies to the individual elements but only to the wrapper span

Reply:
Will have another look today 🤔